src/abstractdataframe/abstractdataframe.jl

```diff
@@ -2261,7 +2261,7 @@ Base.setindex!(::AbstractDataFrame, ::Any, ::Union{Symbol, Integer, AbstractStri
     throw(ArgumentError("syntax df[column] is not supported use df[!, column] instead"))
 
 """
-    reverse(df::AbstractDataFrame)
+    reverse(df::AbstractDataFrame [, start=1 [, stop=nrow(df) ]])
 
 Return a data frame containing the rows in `df` in reversed order.
```

Comment:
If `start` and `stop` are provided, only rows in the `start:stop` range are affected.